scripts/versions.ts

```diff
@@ -1,6 +1,6 @@
-import pkgVersions from 'npm-package-versions';
 import * as semverExtra from 'semver-extra';
 import * as semver from 'semver';
+import pkgVersions = require('npm-package-versions');
```

Comment:
have you tried `import * as pkgVersions from'npm-package-versions';`

Reply:
Yes. It also works, but docs say about this syntax for common js: http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html